Formula/openlibm.rb

```diff
@@ -0,0 +1,22 @@
+class Openlibm < Formula
+  desc "High quality, portable, open source libm implementation"
+  homepage "https://openlibm.org"
+  url "https://github.com/JuliaMath/openlibm/archive/v0.7.0.tar.gz"
+  sha256 "1699f773198018b55b12631db9c1801fe3ed191e618a1ee1be743f4570ae06a3"
+
+  def install
+    lib.mkpath
+    (lib/"pkgconfig").mkpath
+    (include/"openlibm").mkpath
+
+    system "make", "install", "prefix=."
+
+    lib.install Dir["lib/*"].reject { |f| File.directory? f }
+    (lib/"pkgconfig").install Dir["lib/pkgconfig/*"]
+    (include/"openlibm").install Dir["include/openlibm/*"]
+  end
+
+  test do
+    system "make", "test"
```

Comment:
This won't work because the source code is not available in the test environment, which is why `make test` is claiming that the target `test` doesn't exist.

A better test might be to try compiling and linking an example program against the library.
- If you have examples in the source code distribution, you can install them at install time and then copy them into the build environment. For example, this is how the [`libp11`](https://github.com/Homebrew/homebrew-core/blob/55a880ba621d6faa321ed05cf3c6a5d88c07ee45/Formula/libp11.rb#L31) formula does it (the file being built for the test is `auth.c`):
```ruby
def install
  system "./bootstrap" if build.head?
  system "./configure", "--disable-dependency-tracking",
                        "--prefix=#{prefix}",
                        "--with-enginesdir=#{lib}/engines-1.1"
  system "make", "install"
  pkgshare.install "examples/auth.c"
end

test do
  system ENV.cc, "-I#{Formula["openssl@1.1"].include}", "-L#{lib}",
                 "-L#{Formula["openssl@1.1"].lib}", "-lp11", "-lcrypto",
                 pkgshare/"auth.c", "-o", "test"
end
```

- Alternatively, if an example is short enough, it can be included in-place in the formula and written directly into the test environment. Here's how the [`lzlib`](https://github.com/Homebrew/homebrew-core/blob/55a880ba621d6faa321ed05cf3c6a5d88c07ee45/Formula/lzlib.rb#L24) formula does this:
```ruby
test do
  (testpath/"test.c").write <<~EOS
    #include <stdio.h>
    #include <stdint.h>
    #include "lzlib.h"
    int main (void) {
      printf ("%s", LZ_version());
    }
  EOS
  system ENV.cc, "test.c", "-L#{lib}", "-I#{include}", "-llz",
                 "-o", "test"
  assert_equal version.to_s, shell_output("./test")
end
```